lib/src/story_list_tile.dart

```diff
@@ -1,13 +1,16 @@
 import 'package:flutter/material.dart';
 class StoryListTile extends StatelessWidget {
-  const StoryListTile({super.key,required this.storyNumber,required this.storyTitle});
-  final int storyNumber;
-  final String storyTitle;
+  const StoryListTile({super.key,this.number,required this.title});
+  final int? number;
+  final String title;
   @override
   Widget build(BuildContext context) {
-    return ListTile(
-          leading: Text("$storyNumber"),
-          title: Text("Story $storyTitle"),
-        );
+    return InkWell(
```

Comment:
`ListTile` already has an `onTap` property, so let's call that instead of using an `InkWell`. You can add a `void Function()? onTap` property to `StoryListTile` and then just pass it on to `ListTile`